.kokoro/continuous/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ruby:3.1.1 AS build-env
```

Comment:
Might as well use `ruby:3.1.2` or even `ruby:3.1`.

But more to the point, I'm not clear what this dockerfile is for...?

Reply:
I'm not sure about that either 😅  , I took whatever was there in `google-cloud-ruby` . I can remove it if that makes sense.

Reply:
There isn't a `.kokoro/continuous/Dockerfile` in google-cloud-ruby

Reply:
Oh yeah, I just noticed that it was there in my local only. 🤷🏻